### ISSUE TITLE
Baseline survey of surveyors

### ISSUE THREAD

**Issue by doubledubya** [yneggers/SDMCproj#3]
[BACKGROUND]
Create a baseline survey of surveyor characteristics which we need to submit with IRB approval. 

[GOAL]
Surveyor characteristics will be used in balance table to check randomization.

[NOTES]
Joey and I will work on this for the IRB submission. The survey form can be found [here](https://docs.google.com/forms/d/1Vwye2A8GKLdfD3K-RzSWgdl-ebBpcw9x-DUtadxWzPk/edit?usp=sharing), which we anticipate will be administered online through Google Forms.

Shreya can provide feedback when draft version is available and possibly include Hindi translation for final version.

**Comment by yneggers** [comment]
Update draft attached (and in dropbox).

Section 3
	3.1-3.4 – Locus of control (internal vs external choice)
	3.5-3.8 – Propensity for corruption
3.9-3.12 – Reciprocity (negative and positive)
Section 4
4.1-4.16 – Public Service Motivation
Section 5
5.1-5.10 – Big 5 Personality Characteristics
Sectio
[survey_draft_new 180228.docx](https://github.com/yneggers/SDMCproj/files/1772623/survey_draft_new.180228.docx)
n 6
	6.1-6.12 – Raven’s Matrices (cognitive ability)

**Comment by doubledubya** [comment]
@yneggers I went through the document and provided comments. Happy to iterate again after getting your feedback.
Saved over dropbox file (sorry forgot to create another version) [here](https://www.dropbox.com/s/fenm4rqjlqwowrb/survey_draft_new%20180228.docx?dl=0)

**Comment by yneggers** [comment]
@doubledubya Great, just did another round of editing/replying to comments, [here](https://www.dropbox.com/s/shhyrkl2yvjwwda/survey_draft%20180303.docx?dl=0)

**Comment by yneggers** [comment]
@doubledubya Did one more round of replies -- I think we're essentially good to go content-wise, with a bit of formatting to do. For the submission, I'm not sure what Chicago prefers, should we submit the English version (i.e. delete all the Hindi) and then once they approve it, get the Hindi translation incorporated (where we would probably need to submit some form attesting to the accuracy of the translation), or do they want us to do to the translation ahead of time and submit the final version of the survey that would have both included already?

**Comment by doubledubya** [comment]
@yneggers I think we should submit what we have and say that we intend to have it fully translated or are in the process of doing so. I'm not sure whether we should remove all the Hindi, or just remove that Hindi where we know its not the right translation and added new/different content. I lean toward the latter because it shows the intent to translate and that some of it is done already?
I responded to your comment in the baseline survey document. Also, feel free to accept the changes/remove comments if you feel ok about the edit. 

No worries about the deleted comment. I think also mentioned that I went through your comments on the AURA form and if you think the changes look good, I think we're good to submit.

**Comment by yneggers** [comment]
@doubledubya Okay, sounds good on telling them that we will complete/update the translation once the English content is approved, and leaving the Hindi in the survey. I made a few updates to the Hindu version of the consent form (copying over the contact info from the English version) and I accepted the changes you suggested. Otherwise, good to go!  I also made a few updates to the Aura form, which I'm now also good signing off on.
